src/styles/blocks/card/card.scss

```diff
@@ -0,0 +1,58 @@
+.card {
+  line-height: 18px;
+  color: $secondary;
+  padding: 32px 16px 16px;
+  border-radius: 3px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
```

Comment:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order